### DISCUSSION TITLE
Add other component to the toolbar

### DISCUSSION THREAD

**Discussion by foremaple** [vkurko/calendar#273] [Q&A] [answered]
I understand we can add custom buttons and click event to the header toolbar, just wondering what would be the steps to add other control/component to the toolbar, for example, to add a select dropdown, so we can filter the event by changing the selection?

**Comment by vkurko** [comment, marked as answer]
The current version only allows you to add custom buttons. It is now impossible to add anything else, even arbitrary html. There are no plans for implementation yet either. In the future, when moving to Svelte 5, I will try to provide wider support for native mutual integration of Svelte components.

For now, my advice would be to abandon the built-in toolbar, and instead implement a widget completely separate from the calendar with the necessary buttons that will control the calendar through its API.

**Reply by foremaple** [reply]
Thank you @vkurko for your prompt response.
